acceptance/localcluster/localcluster.go

```diff
@@ -426,8 +441,33 @@ func (l *Cluster) Start() {
 	}
 }
 
+// AssertAndStop drains the Events channel and stops the cluster.
+// If the drained events (excluding "start" and "create") don't
+// match the events generated by the operations performed on the
+// nodes in the cluster (restart, kill, ...), the passed Tester
+// receives a fatal error.
+func (l *Cluster) AssertAndStop(t util.Tester) {
```

Comment:
I just factored out `Assert` since it's better to explicitly call it regularly in an acceptance test I'm writing, so I'd like to stick with `AssertAndStop`.